assets/js/app/DataLayer.js

```diff
@@ -112,12 +112,13 @@ LocusZoom.PositionsDataLayer = function(){
 
     LocusZoom.DataLayer.apply(this, arguments);  
     this.id = "positions";
-    this.fields = ["id","position","pvalue","refAllele","ld:state"];
+    this.xfield = "position";
```

Comment:
Datalayer.js hasn't had any variables that are axis/dimension specific, but [Panel.js](https://github.com/statgen/locuszoom/blob/master/assets/js/app/Panel.js) has and they're all of the form `x_foo` or `y_bar` (or `y1_bar` since they tend to refer to axis-related values at the panel level). Examples include `x_axis`, `x_axis_label`, `x_extent`, and `x_scale`. I'd recommend we stick with underscore spacing in variable names where a dimension and an attribute are both present for consistency.